bigquery_etl/metadata/publish_metadata.py

```diff
@@ -41,7 +41,11 @@ def publish_metadata(client, dataset, table, metadata):
         if metadata.description is not None:
             table.description = metadata.description
 
-        table.labels = metadata.labels
+        table.labels = {
+            key: value
+            for key, value in metadata.labels.items()
+            if isinstance(value, str)
```

Comment:
Is there a specific reason to filter stuff out rather than convert to a string?

Reply:
Or at the very least should we log that something has been filtered out?

Reply:
there are too many restrictions on what characters are allowed in labels (doesn't allow `,`, for example). I don't think it's worth to implement any more complex logic just to get these custom labels which aren't likely be looked at anyway